data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSource.java

```diff
@@ -131,7 +131,7 @@ public void start(final Buffer<Record<Log>> buffer) {
             Thread.currentThread().interrupt();
             throw new RuntimeException(ex);
         }
-        LOG.info("Started http source...");
+        LOG.info("Started http source on port " + sourceConfig.getPort() + "...");
```

Comment:
It is best practice to use SLF4J format strings rather than string concatenation.

So this should have been:

```
LOG.info("Started http source on port {}...", sourceConfig.getPort());
```

https://www.slf4j.org/faq.html#logging_performance